vision_agent_tools/models/flux1.py

```diff
@@ -112,75 +115,89 @@ def __call__(
             - max_sequence_length (`int` defaults to 512):
                 Maximum sequence length to use with the `prompt`.
                 to make generation deterministic.
+            - seed (`int`, *optional*): The seed to use for the random number generator.
+                If not provided, a random seed is used.
             - strength (`float`, *optional*, defaults to 0.6):
                 Indicates extent to transform the reference `image`.
                 Must be between 0 and 1.
                 A value of 1 essentially ignores `image`.
-            - seed (`int`, *optional*): The seed to use for the random number generator.
-                If not provided, a random seed is used.
+            - image (Image.Image): The original image to be modified.
+            - mask_image (Image.Image): The mask image indicating areas to be inpainted.
 
         Returns:
             Image.Image | None: The output image if the Flux1 process is successful;
                 None if an error occurred.
         """
 
-        if height % 8 != 0 or width % 8 != 0:
-            raise ValueError(
-                f"`height` and `width` have to be divisible by 8 but are {height} and {width}."
-            )
-
-        if max_sequence_length is not None and max_sequence_length > 512:
-            raise ValueError(
-                f"`max_sequence_length` cannot be greater than 512 but is {max_sequence_length}"
-            )
+        if config.seed is None:
+            config.seed = random.randint(0, 2**32 - 1)
```

Comment:
I'm not sure about modifying the `config`, I think configurations should be immutable.

Reply:
Agreed! Fixed, thanks